### PR TITLE
Fix checkpointing

### PR DESCRIPTION
**Pull request by durch**

### CODE REVIEW

File: common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs [hunk 1]
```diff
@@ -247,24 +247,24 @@ impl DelegatorRewardParams {
     }
 }
 
-#[derive(Debug, Clone, JsonSchema, PartialEq, Serialize, Deserialize, Copy)]
+#[derive(Debug, Clone, JsonSchema, PartialEq, Serialize, Deserialize)]
 pub struct StoredNodeRewardResult {
     reward: Uint128,
-    lambda: Uint128,
-    sigma: Uint128,
+    lambda: String,
```
Comment (jstuczyn) [inline common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs]: just leaving a comment here to remember to [presumably] revert it back to Uint128 before merging

Comment (durch) [reply common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs]: No, no, leave it as string, Uint128 will truncate the float

Comment (durch) [reply common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs]: Never mind, I see what you did there, will test it out in QA

File: contracts/mixnet/src/constants.rs [hunk 2]
```diff
@@ -1,9 +1,8 @@
 // Copyright 2022 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: Apache-2.0
 
-// approximately 1 week (assuming 5s per block)
-// i.e. approximately quarter of the interval (there are 3600 * 60 * 7 = 604800 seconds in a week, i.e. ~604800 / 5 = 120960 blocks)
-pub const MINIMUM_BLOCK_AGE_FOR_REWARDING: u64 = 120960;
+// approximately 1 epoch (assuming 5s per block)
+pub const MINIMUM_BLOCK_AGE_FOR_REWARDING: u64 = 720;
```
Comment (jstuczyn) [inline contracts/mixnet/src/constants.rs:5]: another note to remember to revert that before merging : )

Comment (durch) [reply contracts/mixnet/src/constants.rs:5]: I don't think we need to revert this, if they get in the active set they should be rewarded

**Pull request merged**